platforms/linux-ppc64le/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:bullseye
+LABEL maintainer="Lovell Fuller <npm@lovell.info>"
+
+# Create Debian 10 (glibc 2.28) container suitable for cross-compiling Linux s390x binaries
+
+# Path settings
+ENV \
+  RUSTUP_HOME="/usr/local/rustup" \
+  CARGO_HOME="/usr/local/cargo" \
+  PATH="/usr/local/cargo/bin:$PATH"
+
+# Build dependencies
+RUN \
+  apt-get update && \
+  apt-get install -y curl && \
+  dpkg --add-architecture ppc64el && \
+  apt-get install -y \
+    autoconf \
+    autopoint \
+    cmake \
+    crossbuild-essential-ppc64el \
+    gettext \
+    git \
+    gperf \
+    jq \
+    libtool \
+    ninja-build \
+    pkg-config \
+    python3-packaging \
+    python3-pip \
+    && \
+  curl https://sh.rustup.rs -sSf | sh -s -- -y \
+    --no-modify-path \
+    --profile minimal \
+    && \
+  rustup target add powerpc64le-unknown-linux-gnu && \
+  pip3 install meson
+
+# Handy for debugging the compiled targets in Highway (hwy_list_targets)
+#RUN apt-get install -y qemu-user-static
+#ENV QEMU_LD_PREFIX="/usr/s390x-linux-gnu"
+
+# Compiler settings
+ENV \
+  PKG_CONFIG="powerpc64le-linux-gnu-pkg-config --static" \
+  PLATFORM="linux-ppc64le" \
+  CHOST="powerpc64le-linux-gnu" \
+  RUST_TARGET="powerpc64le-unknown-linux-gnu" \
+  FLAGS="" \
+  WITHOUT_HIGHWAY="false" \
```

Comment:
This still disables Highway, since this env variable is checked with `-z`.
https://github.com/lovell/sharp-libvips/blob/ad960df07ac72b911086a514683dbd346038ff3f/build/lin.sh#L320

```suggestion
```